src/components/Search/SearchRouter/SearchRouter.tsx

```diff
@@ -84,10 +84,13 @@ function SearchRouter({onRouterClose}: SearchRouterProps) {
     }, [debouncedInputValue, searchOptions]);
 
     const recentReports: OptionData[] = useMemo(() => {
-        const currentSearchOptions = debouncedInputValue === '' ? searchOptions : filteredOptions;
-        const reports: OptionData[] = [...currentSearchOptions.recentReports, ...currentSearchOptions.personalDetails];
-        if (currentSearchOptions.userToInvite) {
-            reports.push(currentSearchOptions.userToInvite);
+        if (debouncedInputValue === '') {
+            return searchOptions.recentReports.slice(0, 10);
```

Comment:
I'm curious if that doesn't change previous logic.

Originally we had `debouncedInputValue === '' ? searchOptions` - so it meant that if the input is empty, we push in the `searchOptions` (so I guess contextual search?).

But now we only show recent reports? Is that the change we want?

Reply:
Contextual search is added independently earlier in the code. This block of code was only responsible for adding recent chats section(which consisted of **recentReports**, **personalDetails** and **userToInvite**). Right now in case when text input is empty, we are limiting the recent chats section to only **recentReports**.